tests/integration/rdb.tcl

```diff
@@ -231,6 +231,7 @@ set system_name [string tolower [exec uname -s]]
 if {$system_name eq {linux}} {
 
 start_server {overrides {save ""}} {
+    if {[r debug mallctl arenas.page] == 4096} {
```

Comment:
we can't use that trick without also verifying that we use jemalloc and not another allocator.
we already skip this test on non-linux platforms, let's add a call to `[exec getconf PAGESIZE]` in the linux check above.